accepted/2020/or-greater-defines/or-greater-defines.md

```diff
@@ -0,0 +1,218 @@
+# OR_GREATER preprocessor symbols for TFMs
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+Today, projects automatically define `#if` symbols based on the friendly TFM
+representation. This includes both versionless- as well as version-specific
+symbols. For .NET 6, we plan to add `XXX_OR_GREATER` variants that are
+accumulative in nature:
+
+TFM                 | Existing | New
```

Comment:
```suggestion
TFM                 | Existing | Additional
```

The word "New" here is ambiguous between "this is the new defines we generate that replace the old ones" and "these are the new defines in addition to the older ones". Rest of the doc clears that up but think it could help to be more explicit here.

Reply:
Reworded to `Proposed Additions`